### PR TITLE
Add requirements.txt 

### PR DESCRIPTION
**Pull request by yihong0618**
Some test failed I don't know why.
e.g.
https://github.com/yihong0618/nebula-python/actions/runs/855718226

### CODE REVIEW

**Comment by yihong0618** [comment]
https://github.com/vesoft-inc/nebula-python/pull/104

File: README.md [hunk 1]
```diff
@@ -60,6 +60,15 @@ When your environment cannot access `pypi`, you need to install the following pa
 - httplib2
 - futures   # python2.x is needed
```
Comment (laura-ding) [inline README.md:61]: line 57-61 can be deleted, directly to use requirements.txt is ok.

Comment (yihong0618) [reply README.md]: Done

**Comment by laura-ding** [comment]
> Some test failed I don't know why.
> e.g.
> https://github.com/yihong0618/nebula-python/actions/runs/855718226

Maybe the errorcode change of server, i need to update the thrift file

**Comment by yihong0618** [comment]
> > Some test failed I don't know why.
> > e.g.
> > https://github.com/yihong0618/nebula-python/actions/runs/855718226
> 
> Maybe the errorcode change of server, i need to update the thrift file

Got it

**Comment by yihong0618** [comment]
> > Some test failed I don't know why.
> > e.g.
> > https://github.com/yihong0618/nebula-python/actions/runs/855718226
> 
> Maybe the errorcode change of server, i need to update the thrift file

FYI

Its strange that  3.7 passed sometimes 3.6 failed

image

**Comment by laura-ding** [comment]
> > > Some test failed I don't know why.
> > > e.g.
> > > https://github.com/yihong0618/nebula-python/actions/runs/855718226
> > 
> > 
> > Maybe the errorcode change of server, i need to update the thrift file
> 
> FYI
> 
> Its strange that 3.7 passed sometimes 3.6 failed
> 
> image

because the leader of metad is changed. The randomly selected Meta happens to be the leader, so it succeeds

**Review by laura-ding — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**